chips/sam4l/src/scif.rs

```diff
@@ -333,7 +376,52 @@ pub unsafe fn setup_pll_osc_48mhz() {
     );
 
     // Wait for the PLL to become locked
-    while (*SCIF).pclksr.matches_all(Interrupt::PLL0LOCK::CLEAR) {}
+    while !(*SCIF).pclksr.is_set(Interrupt::PLL0LOCK){}
+}
+
+pub unsafe fn disable_pll() {
+    unlock(Register::PLL0);
+    (*SCIF).pll0.modify(PllControl::PLLEN::CLEAR);
```

Comment:
This `modify` will have the same problem as @bradjc noticed in #929. Even if the lock register doesn't seem to be doing anything in practice, I think it best to still use it as required by the datasheet.